src/mongoserver.py

```diff
@@ -53,18 +52,19 @@ def check_server_info(self, client: MongoClient):
         """
         return client.server_info()
 
-    def is_ready(self, as_stand_alone=False) -> bool:
+    def is_ready(self, standalone=False) -> bool:
```

Comment:
```suggestion
    def is_ready(self, standalone: bool=False) -> bool:
```

other methods seem to have type annotations, so might be worth being consistent here. Also, on a positive side if you indicate the type with an annotation, you can simplify the docstring and remove the type information :)